src/components/Search/Filter.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import 'react-native'
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react-native'
+
+import SearchFilter, { FilterButton } from './Filter'
+
+const availableFilters = [
+  'First Letters',
+  'Main Letters',
+  'Bani',
+  'Translation',
+  'Pronunciation',
+]
+
+describe( '<SearchFilter />', () => {
+  it( 'should click a filter button', () => {
```

Comment:
`<SearchFilter />` `should click a filter button`?

I think you probably meant `should fire the onPress event, given a press` perhaps?

Reply:
etc for the test names below